cmd/client-admin.go

```diff
@@ -79,11 +79,11 @@ func newAdminFactory() func(config *Config) (*madmin.AdminClient, *probe.Error)
 			var transport http.RoundTripper = &http.Transport{
 				Proxy: http.ProxyFromEnvironment,
 				DialContext: (&net.Dialer{
-					Timeout:   30 * time.Second,
-					KeepAlive: 30 * time.Second,
+					Timeout:   60 * time.Second,
+					KeepAlive: 60 * time.Second,
```

Comment:
increasing keep alive is also not needed since 30secs means that its `keepAlive` probe timeout which is usually 30 * 9 (net.ipv4.tcp_keepalive_probes)  i.e almost 4.5minutes not sure if we need to keep alive more than that.